packages/did/src/DidDetails/LightDidDetails.ts

```diff
@@ -94,43 +93,4 @@ export class LightDidDetails extends DidDetails {
       this.keyRelationships.keyAgreement = [`${this.didUri}#encryption`]
     }
   }
-
```

Comment:
Removed as I realized they are already exposed by the `DidDetails` abstract superclass.